plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -358,6 +479,73 @@ bool CppParser::parse()
   return success;
 }
 
+void CppParser::initBuildActions()
+{
+  (util::OdbTransaction(_ctx.db))([&, this] {
+    for (const model::BuildAction& ba : _ctx.db->query<model::BuildAction>())
+      _parsedCommandHashes.insert(util::fnvHash(ba.command));
+  });
+}
+
+void CppParser::markAsModified(const model::FilePtr file_)
+{
+  auto inclusions = _ctx.db->query<model::CppHeaderInclusion>(
+    odb::query<model::CppHeaderInclusion>::included == file_->id);
+
+  for (auto inc : inclusions)
+  {
+    model::FilePtr loaded = inc.includer.load();
+    if(_ctx.fileStatus.count(loaded->path) == 0)
+    {
+      _ctx.fileStatus.insert(std::make_pair(loaded->path, IncrementalStatus::MODIFIED));
+      LOG(debug) << "File modified: " << loaded->path;
+
+      markAsModified(loaded);
+    }
+  }
+}
+
+std::set<model::CppNodeId> CppParser::collectNodeSet(model::CppNodeId node_) const
```

Comment:
What's a node set? Also, spaces and general formatting mismatch in this code too.

Reply:
Not needed after all, removed in 1acae06bb70d50831c4887cfad6ef9df459b99cb.